duckduckgo/iOS
# Add app version&build number to Pixels

Task/Issue URL: https://app.asana.com/0/856498667320406/1150186272067857
Tech Design URL:
CC:

**Description**:
Move extension to Core package, include version&build number in Pixels as a parameter.

**Steps to test this PR**:
- See that `appVersion` is added as a parameter to Pixel requests.
- Ensure that user agent has not been affected by the change.


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
